src/lib/toolbar/toolbar.md

```diff
@@ -49,3 +49,7 @@ easily accomplished with `display: flex`:
 The color of a `<md-toolbar>` can be changed by using the `color` property. By default, toolbars
 use a neutral background color based on the current theme (light or dark). This can be changed to 
 `'primary'`, `'accent'`, or `'warn'`.  
+
+### Accessibility
+Toolbar items without text or labels should be given a meaningful label via `aria-label` or
```

Comment:
I think this isn't quite right. Similar to `list` and `grid-list`, we treat toolbar as decorative-only by default. Generally, the toolbar is used as a header where `role="header"` would be appropriate. 

Only if the use-case of the toolbar match that of [`role="toolbar"`](https://www.w3.org/TR/wai-aria/roles#toolbar), the user should add the role and an appropriate label.